custom-vu/src/main/kotlin/jces1209/vu/page/DcIssueNavigator.kt

```diff
@@ -0,0 +1,53 @@
+package jces1209.vu.page
+
+import jces1209.vu.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.ExpectedConditions.*
+import java.time.Duration
+import java.util.*
+
+
+class DcIssueNavigator(
+    driver: WebDriver
+) {
```

Comment:
It looks like there are a lot of the same things for DC and Cloud
Please extract them to the parent class and reuse